Google Cloud/google-report/_meta/smart-descriptions.json

```diff
@@ -350,6 +350,170 @@
       }
     ]
   },
+  {
```

Comment:
If you apply the suggestion on `event.type` and `event.category`, I think you can simplify the smart-descriptions according these two fields in the conditions.
Otherwise, a smart-description is missing for `DEVICE_BOOT_STATE_CHANGE`